services/charts_updater.go

```diff
@@ -801,67 +793,41 @@ func performanceDistribution365dChartData() (*types.GenericChartData, error) {
 }
 
 func depositsChartData() (*types.GenericChartData, error) {
-	var err error
-
-	eth1Rows := []struct {
-		Timestamp int64
-		Amount    uint64
-		Valid     bool
-	}{}
-
-	err = db.ReaderDb.Select(&eth1Rows, `
-		select
-			extract(epoch from block_ts)::int as timestamp,
-			amount,
-			valid_signature as valid
-		from eth1_deposits
-		order by timestamp`)
-	if err != nil {
-		return nil, fmt.Errorf("error getting eth1-deposits: %w", err)
+	if LatestEpoch() == 0 {
```

Comment:
This is strange, as a few lines below, we check again and reduce by one. We shouldn't call the function twice and we should always the same value / calculation.

Reply:
Same situation appears again.

Reply:
fixed